protocol/msgpack/options.go

```diff
@@ -21,12 +21,13 @@
 package msgpack
 
 import xpool "github.com/m3db/m3x/pool"
+import "math"
```

Comment:
Group them into an import block?

Reply:
Not sure why VS Code did it like that lol